release-notes/9.0/patch-release-index.json

```diff
@@ -0,0 +1,41 @@
+{
+  "channel-version": "9.0",
```

Comment:
Is it worth having an explicit `latest-version` or similar property to act as a key into `releases`, rather than relying on `releases[0]` always being the latest?